python/lsst/rucio/register/script.py

```diff
@@ -113,6 +126,7 @@ def main():
     "-C", "--rucio-register-config", required=False, type=str, help="configuration file used for registration"
 )
 @click.option("-D", "--debug", required=False, is_flag=True, help="set loglevel to DEBUG")
+@click.option("-i", "--info", required=False, is_flag=True, help="set loglevel to INFO")
```

Comment:
As **click** does not support mutually exclusive options (not without a [third-party package](https://github.com/click-contrib/click-option-group) anyway) you may consider letting the user to set up the log level explicitly with something like a [choice option](https://click.palletsprojects.com/en/stable/options/#choice-options) if changing command line interface is still a possibility. But I'm leaving the decision up to you.

Reply:
In pipetask it sets up logging with
```python
@click.command(cls=PipetaskCLI, context_settings=dict(help_option_names=["-h", "--help"]))
@log_level_option()
@long_log_option()
@log_file_option()
@log_tty_option()
@log_label_option()
```
with a log initializer called internally.

Reply:
Added @log_level_option